tests/test_shortcuts.py

```diff
@@ -219,6 +241,7 @@ def test_other_providers():
     assert swap_autosuggestion_down(event) is None
 
 
+@pytest.mark.asyncio
```

Comment:
If you have time (or as a separate issue/PR), it would be awesome to:

 1. mark pytest.mark.anyio (to make sure we test on asyncio and trio)
 2. make it an error at least on CI to have an async test skipped.

Reply:
I will do so in another PR

> mark pytest.mark.anyio (to make sure we test on asyncio and trio)

I guess also here?

https://github.com/ipython/ipython/blob/46b09de968521d08a2903316090557da547007ee/tests/test_magic.py#L1248-L1268

> make it an error at least on CI to have an async test skipped.

agree a good idea, I guess we could grep pytest output - or any other ideas?

Reply:
I was hopping there was a pytest option...